cdap-docs/vars

```diff
@@ -68,7 +68,7 @@ GIT_BRANCH_CDAP_GUIDES="develop"
 # Used by integrations-manual
 GIT_BRANCH_CDAP_PACKS="develop"
 
-GIT_CASK_MARKET_VERSION="v2"
+GIT_HUB_VERSION="v2"
```

Comment:
eesh. this variable becomes really misleading now. not sure we can do anything though.

Reply:
oh lol... I finally see how it is misleading "git_hub"...

Reply:
yea, the GIT prefix is needed based, since that prefix is used for all these variables.